sonic-chassisd/scripts/chassisd

```diff
@@ -92,6 +96,8 @@ INVALID_IP = '0.0.0.0'
 CHASSIS_MODULE_ADMIN_STATUS = 'admin_status'
 MODULE_ADMIN_DOWN = 0
 MODULE_ADMIN_UP = 1
+REBOOT_CAUSE_DIR = "/host/reboot-cause/module/"
```

Comment:
@rameshraghupathy rename to `MODULE_REBOOT_CAUSE_DIR` ?

Reply:
@prgeor Done